cinema/permissions.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework.permissions import BasePermission, SAFE_METHODS
+
+
+class IsAdminOrIfAuthenticatedReadOnly(BasePermission):
+
+    def has_permission(self, request, view):
+        return bool(
+            (request.method in SAFE_METHODS
+             and request.user
+             and request.user.is_authenticated
+             )
```

Comment:
Reformat your code, when you have arguments on the line with opening parenthesis, closing parenthesis should have arguments as well and vice versa. Pick up one approach but do not mix them.